web-app/src/app/leftpanel/leftpanel.component.ts

```diff
@@ -14,21 +14,23 @@ export class LeftpanelComponent implements OnInit {
   packet: Packet;
 
   heartBeat: Boolean;
+  timestamp: Date;
   // inject the PacketService
   constructor(private packetService: PacketService, private heartbeatService: HeartbeatService) {
   }
 
   ngOnInit() {
     // initialize with default values
     this.packet = this.packetService.getData();
-
+    this.timestamp = new Date(Number(this.packet.timestamp))
     // observe changes and update public variable when changed
     // note the dollar sign, this means you can subscribe to the object
     // see _services/packet.service.ts
```

Comment:
the comments for my timestamp are also here

Reply:
Did you want to delete them?